R/db.R

```diff
@@ -134,7 +134,7 @@ dbWriteTableUntilSuccess <- function(dbconn, tblname, df, append = FALSE, row.na
 #' @param df data.frame. The data to write.
 #' @param key character. The identifier column name.
 #' @inheritParams cache
-write_data_safely <- function(dbconn, tblname, df, key, safe_columns) {
+write_data_safely <- function(dbconn, tblname, df, key, safe_columns, blacklist) {
```

Comment:
This is probably the only function (besides `cache`) that actually requires a `blacklist` parameter (also, the documentation needs to be updated -- not sure how this passed tests).